data/_models/shared/Organisations.csv

```diff
@@ -1,15 +1,15 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
 Organisations,,,,,,,,,,,"org:Organization,foaf:Agent",Research departments and research groups,"DataCatalogueFlat,EMA",
 Organisations,,overview,heading,,,,,,,,,General information,DataCatalogueFlat,
-Organisations,,id,,1,true,,,,,,dcterms:identifier,Internal identifier,"DataCatalogueFlat,EMA",
-Organisations,,pid,,2,,,,,,,dcterms:identifier,Persistent identifier,"DataCatalogueFlat,EMA",
-Organisations,,acronym,,,,,,,,,dcterms:identifier,Acronym if applicable,"DataCatalogueFlat,EMA",
-Organisations,,name,text,3,true,,,,,,"dcterms:title,foaf:name",Name used in European projects,"DataCatalogueFlat,EMA",
-Organisations,,type,ontology_array,,,CatalogueOntologies,Organisation types,,,,,Type of organisation; in which sector is the organisation active?,"DataCatalogueFlat,EMA",
+Organisations,,id,,1,true,,,,,,dct:identifier,Internal identifier,"DataCatalogueFlat,EMA",
+Organisations,,pid,,2,,,,,,,dct:identifier,Persistent identifier,"DataCatalogueFlat,EMA",
+Organisations,,acronym,,,,,,,,,skos:altLabel,Acronym if applicable,"DataCatalogueFlat,EMA",
+Organisations,,name,text,3,true,,,,,,"dct:title,skos:prefLabel",Name used in European projects,"DataCatalogueFlat,EMA",
```

Comment:
foaf:name, see: https://health-ri.atlassian.net/wiki/spaces/FSD/pages/121110529/Core+Metadata+Schema+Specification#Agent

Reply:
> If the Agent is an organisation, the use of the [Organization Ontology](https://www.w3.org/TR/vocab-org/) is recommended.

per the table underneath [Mandatory classes](https://health-ri.atlassian.net/wiki/spaces/FSD/pages/121110529/Core+Metadata+Schema+Specification#Mandatory-Classes)

Reply:
We want to move to ROR. Moreover, I want to try and resist adding columns just because of dcat. First think of users, then of standards.